src/Button/Button.js

```diff
@@ -0,0 +1,32 @@
+import './button.scss';
+
+export class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeClass: ''
+    };
+  }
+  clickHandle = () => {
+    if (this.state.activeClass) {
+      this.setState({ activeClass: '' });
+    } else { this.setState({ activeClass: 'active' }); }
+    return null;
+  }
+  render() {
+    return (
+      <div className="text-holder">
+        <button
+          onClick={this.clickHandle}
+          className={this.state.activeClass}
+        > {!this.state.activeClass ? 'show' : 'hide'}
```

Comment:
why here you are using 'show' and 'active', but in state you're using '' and 'active'. In this case you need state.activeClass as boolean.

Reply:
done